Convert Python license classifiers to SPDX expressions (#32)
befeleme/pyp2spec: CHANGELOG.md

# Changelog

# Unreleased
### Changed

> - Convert the Python classifiers to Fedora SPDX expressions (where possible, based on PEP-0639)

Reviewer: ```suggestion
- Convert the Python classifiers to Fedora SPDX expressions (where possible, based on [PEP 639](https://peps.python.org/pep-0639/))
```

> - When `--fedora-compliant` option is set, a check of the converted expressions is performed
> to determine license is good for Fedora. Data is provided by the [Fedora Legal team](https://gitlab.com/fedora/legal/fedora-license-data/)

Reviewer: ```suggestion
- When `--fedora-compliant` option is set, a check of the converted license expressions is performed
to determine whether the license is allowed in Fedora. Data is provided by the [Fedora Legal team](https://gitlab.com/fedora/legal/fedora-license-data/)
```
